src/rollupToVinyl.ts

```diff
@@ -17,39 +17,45 @@ function isEmittedAsset(
 }
 
 export default function rollupToVinyl(
-  outputDir: string,
   inputOptions: rollup.RollupOptions,
   outputOptions: rollup.OutputOptions,
 ) {
   const stream = new Readable({ objectMode: true });
   stream._read = () => {};
 
+  function generatePath(fileName: string) {
```

Comment:
shouldn't we have functions declared outside other functions? seems more readable

Reply:
That makes sense if it's not done for any reason, but in this case it uses part of the outputOptions parameter in the function body.